spanner/spannertest/funcs.go

```diff
@@ -52,6 +52,14 @@ var functions = map[string]function{
 			return strings.HasPrefix(s, prefix), spansql.Type{Base: spansql.Bool}, nil
 		},
 	},
+	"LOWER": {
```

Comment:
I don't think this change belongs in this PR.

Reply:
Can move it to a separate one, just smuggled it in here as I was testing with it

Reply:
Ah, ok. I would prefer to have it in a separate PR, unless it is actually used in one of the test cases (I don't think it is at this moment, right?)